src/plugins/intel_gpu/src/graph/include/program_node.h

```diff
@@ -301,6 +301,10 @@ struct program_node {
     bool can_be_optimized() const { return optimized; }
     void can_be_optimized(bool opt) { optimized = opt; }
 
+    // check/set if the node is runtime skippable
+    bool is_runtime_skippable() const { return runtime_skippable; }
+    void is_runtime_skippable(bool skippable) { runtime_skippable = skippable; }
```

Comment:
Minor change, but how about change this to "set_runtime_skippable" to reduce confusion of the purpose of the function?

Reply:
Done